.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Describe your issue
+
+(Tell us what the problem is in a few word)
+
+# Your environment
+
+Please provide the following informations:
+ - Current Inmanta version: 
+ - Operating System: 
+ - Python version:
+ - PostgreSQL version:
+
+Using Linux? You can replace the list by the output of the following command:  
+echo " - $(inmanta --version)"; \
+echo " - OS: $(source /etc/os-release; echo $PRETTY_NAME)"; \
+echo " - Python version: $(python3 --version)"; \
+echo " - PostreSQL version: $(psql --version)"
+
+# Steps to reproduce
+
+1. step 1
+2. step 2
+3. you get it...
+
+# Expected behavior
+
+(Tell us what you expected this list of steps to do)
+
+# Actual behavior
+
+(Tell us what actually happened)
```

Comment:
A bit redundant wrt to Steps to reproduce I think?

Reply:
In https://github.com/inmanta/inmanta/pull/2416/commits/97df07c2c9d53836b23046f865629bacef6439c7 I remove this section and add one more step in *steps to reproduce* to suggest that it would go there, is this alright?